packages/test-helpers/lib/web3.js

```diff
@@ -0,0 +1,130 @@
+module.exports = web3 => {
+
+  const GETH_NODE_ID = 'Geth'
+  const GANACHE_NODE_ID = 'TestRPC'
+
+  const isGeth = () => web3.version.node.includes(GETH_NODE_ID)
+
+  const isGanache = () => web3.version.node.includes(GANACHE_NODE_ID)
+
+  const advanceBlock = async () => {
+    await web3.currentProvider.send({
+      jsonrpc: '2.0',
+      method: 'evm_mine',
+      id: new Date().getTime()
+    })
+    return web3.eth.getBlock('latest').hash
```

Comment:
We almost definitely need to use an `await` on `getBlock()` first before accessing `hash`

Reply:
Ouch! Good catch